src/net.cpp

```diff
@@ -1103,12 +1103,9 @@ void CConnman::DisconnectNodes()
             if (pnode->GetRefCount() <= 0) {
                 bool fDelete = false;
                 {
-                    TRY_LOCK(pnode->cs_inventory, lockInv);
-                    if (lockInv) {
-                        TRY_LOCK(pnode->cs_vSend, lockSend);
-                        if (lockSend) {
-                            fDelete = true;
-                        }
+                    TRY_LOCK(pnode->cs_vSend, lockSend);
+                    if (lockSend) {
+                        fDelete = true;
```

Comment:
From commit message
> Therefore, cs_inventory can never be locked when the TRY_LOCK(cs_inventory) is reached in DisconnectNodes(). Remove the TRY_LOCK.

I understand that `if (lockInv)` is never true, and the code block `TRY_LOCK(pnode->cs_vSend, lockSend); ...` is dead in master. This commit makes this code always reachable, right?

Reply:
I think the comment says "Therefore, cs_inventory can never be locked *by another thread* ...", so it is always true (success).

Reply:
Sorry, you're right that this commit message is unclear. Marco is correct that I meant "cs_inventory can never be locked by another thread". I'll update the commit message.

Reply:
I've made the commit log clearer.